src/nbt/compound.rs

```diff
@@ -50,6 +50,32 @@ impl NbtCompound {
         bytes.freeze()
     }
 
+    pub fn from_values(values: Vec<(&str, NbtTag)>) -> Self {
```

Comment:
Why random `&str` here?

Reply:
Oops, this shouldn't be here as it was removed in favor of FromIter, it somehow survived in my stash and got in here by accident